git-transport/src/client/blocking_io/http/reqwest.rs

```diff
@@ -210,16 +210,7 @@ mod remote {
     #[derive(Default, Clone)]
     pub struct Config {
         pub config: Option<String>,
-        pub configure: Option<
-            Arc<
-                Mutex<
-                    dyn FnMut(&mut reqwest::blocking::ClientBuilder) -> reqwest::blocking::ClientBuilder
-                        + Send
-                        + Sync
-                        + 'static,
-                >,
-            >,
-        >,
+        pub configure_request: Option<Arc<Mutex<dyn FnMut(&mut reqwest::blocking::Request) + Send + Sync + 'static>>>,
```

Comment:
For my use case I need to configure `Request` instead of `ClientBuilder` so I made changes to only support that.